packages/components/src/date-time/stories/time-input.story.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+/**
+ * Internal dependencies
+ */
+import { TimeInput } from '../time-input';
+
+const meta: Meta< typeof TimeInput > = {
+	title: 'Components/TimeInput',
+	component: TimeInput,
+	argTypes: {
+		onChange: { action: 'onChange', control: { type: null } },
+	},
```

Comment:
```suggestion
	argTypes: {
		onChange: { action: 'onChange', control: { type: null } },
		value: { control: { type: null } },
	},
```

We've usually been `null`-ing out the Storybook control for the `value` prop. In this case I think it's especially good because when the `value` control can take arbitrary user-generated values, any out-of-bounds value (let's say `100`) can be rendered directly into the input field for the component, and that's weird 😅 And it's overkill to add value-clamping logic before it hits the input's own min/max validation.

Reply:
Agree! 👍